cdap-ranger/cdap-ranger-binding/src/main/java/co/cask/cdap/security/authorization/ranger/binding/RangerAuthorizer.java

```diff
@@ -0,0 +1,306 @@
+    //TODO: Investigate if its possible to make the enforce call with set of actions rather than one by one
+    for (Action action : set) {
+      enforce(entityId, principal, action);
+    }
+  }
+
+  @Override
+  public void grant(EntityId entity, Principal principal, java.util.Set<Action> actions) throws Exception {
+    LOG.warn("Grant operation not supported by Ranger for CDAP");
+  }
+
+  @Override
+  public void revoke(EntityId entity, Principal principal, java.util.Set<Action> actions) throws Exception {
+    LOG.warn("Revoke operation not supported by Ranger for CDAP");
+  }
+
+  @Override
+  public void revoke(EntityId entity) throws Exception {
+    LOG.warn("Revoke for entity operation not supported by Ranger for CDAP");
+  }
+
+  @Override
+  public void createRole(Role role) throws Exception {
+    LOG.warn("Create role operation not supported by Ranger for CDAP");
+
+  }
+
+  @Override
+  public void dropRole(Role role) throws Exception {
+    LOG.warn("Drop role operation not supported by Ranger for CDAP");
+
+  }
+
+  @Override
+  public void addRoleToPrincipal(Role role, Principal principal) throws Exception {
+    LOG.warn("Add role to principal operation not supported by Ranger for CDAP");
+
+  }
+
+  @Override
+  public void removeRoleFromPrincipal(Role role, Principal principal) throws Exception {
+    LOG.warn("Remove role from principal operation not supported by Ranger for CDAP");
+
+  }
+
+  @Override
+  public Set<Role> listRoles(Principal principal) throws Exception {
+    LOG.warn("List roles operation not supported by Ranger for CDAP");
+    return null;
+  }
+
+  @Override
+  public Set<Role> listAllRoles() throws Exception {
+    LOG.warn("List all roles operation not supported by Ranger for CDAP");
+    return null;
+  }
+
+  @Override
+  public Set<Privilege> listPrivileges(Principal principal) throws Exception {
+    LOG.warn("List privileges operation not supported by Ranger for CDAP");
+    return null;
+  }
+
+  private String getRequestingUser() throws IllegalArgumentException {
+    Principal principal = context.getPrincipal();
+    return principal.getName();
+  }
+
+  private String toRangerAccessType(Action action) {
+    return action.toString().toLowerCase();
+  }
+
+  /**
+   * Sets the access resource appropriately depending on the given entityId
+   *
+   * @param entityId the entity which needs to be set to
+   * @param rangerAccessResource the {@link RangerAccessResourceImpl} to set the entity values to
+   */
+  private void setAccessResource(EntityId entityId, RangerAccessResourceImpl rangerAccessResource) {
+    EntityType entityType = entityId.getEntityType();
+    switch (entityType) {
+      case INSTANCE:
+        rangerAccessResource.setValue(KEY_INSTANCE, ((InstanceId) entityId).getInstance());
+        break;
+      case NAMESPACE:
+        rangerAccessResource.setValue(KEY_INSTANCE, instanceName);
```

Comment:
Should we put instanceid as the value here since `InstanceId` is not a child of `NamespaceId`?